acceptance/bundle/deploy/experimental-python/test.toml

```diff
@@ -0,0 +1,2 @@
+Local = true
+Cloud = false # need deterministic ids and clean workspace
```

Comment:
Why do you need "deterministic ids and clean workspace"?

Check out how we use $UNIQUE_NAME in other tests.

Reply:
We snapshot the output of "databricks jobs list". We can possibly do some post-processing and filtering, but I'm not sure it will be very reliable if the workspace has other jobs. If we want a cloud test, I would write the test scenario differently.

Reply:
Instead of listing jobs, this could do a GET on the job that was deployed instead.

Reply:
We can make it work, but I don't see what we get by running these tests against a real cloud. I think it wouldn't hurt to run jobs created through templates, but we also can't do that on every PR.